shell/artifacts/Social/Social.recipes

```diff
@@ -54,9 +54,10 @@ recipe
     people <- people
 
 recipe
-  map #BOXED_posts as posts
-  map #BOXED_avatar as avatars
+  map #BOXED_stats as posts
   // TODO(wkorman): Merge together posts and stats with CopyCollection if needed.
+  // map #BOXED_posts as posts
```

Comment:
This switches this particle over to stats, with the TODO to combine them and bring posts back?

Reply:
Yeah, intentional, since that's what I tested last and the next step is to combine them so it was fine either way.